### PR TITLE
xpCNF3HW: publish non-production DCS public certificates

### PR DESCRIPTION
**Pull request by bjgill**
## Why

New DCS clients will need these certificates in order to establish a connection to the DCS. Include them in the pilot docs to make things easier for clients and reduce the DCS team's support burden.

Also tweak some certificate/key wording for consistency/accuracy.

## What

Add the two public certificates and the CA bundle. Add links to these from appropriate places.

## How to review

Run the preview and confirm you can download the files from the appropriate links.

### CODE REVIEW

**Comment by bjgill** [comment]
Monday is my NWD - feel free to merge this in my absence

File: source/integrating-with-the-dcs.html.md.erb [hunk 1]
```diff
@@ -27,6 +27,7 @@ Your client will need to be able to sign and encrypt JSON data and make HTTP req
 To build a working DCS client, you'll need to:
 
 1. [Generate keys and request certificates](/generate-keys-and-request-certificates/#generate-keys-and-request-certificates) - you’ll [contact DCS support](/support/#support) to get the Public Key Infrastructure (PKI) information so you can get started with integrating with the DCS.
+1. Download the [DCS public encryption certificate], [DCS public signing certificate], and [DCS Certificate Authority (CA) certificate bundle].
 1. [Create a network connection to the DCS](/create-a-network-connection/#create-a-network-connection-to-the-dcs) using mutual authentication.
 1. [Set up your JOSE certificate thumbprints](/Set-up-your-JOSE-certificate-thumbprints/#set-up-your-jose-certificate-thumbprints).
 1. [Sign and encrypt a DCS payload](/sign-and-encrypt-a-DCS-payload/#sign-and-encrypt-a-dcs-payload).
```
Comment (PippaClarkGDS) [inline source/integrating-with-the-dcs.html.md.erb:33]:
```suggestion
1. [Generate keys and request certificates](/generate-keys-and-request-certificates/#generate-keys-and-request-certificates) - you’ll [contact DCS support](/support/#support) to get the Public Key Infrastructure (PKI) information so you can get started with integrating with the DCS.
1. Download the [DCS public encryption certificate].
1. Download the [DCS public signing certificate].
1. Download the [DCS Certificate Authority (CA) certificate bundle].
1. [Create a network connection to the DCS](/create-a-network-connection/#create-a-network-connection-to-the-dcs) using mutual authentication.
1. [Set up your JOSE certificate thumbprints](/Set-up-your-JOSE-certificate-thumbprints/#set-up-your-jose-certificate-thumbprints).
1. [Sign and encrypt a DCS payload](/sign-and-encrypt-a-DCS-payload/#sign-and-encrypt-a-dcs-payload).
```

Comment (PippaClarkGDS) [reply source/integrating-with-the-dcs.html.md.erb:33]: this numbered list was built to reflect the H1s of the other pages. Would it make more sense for 3 new pages to be created where users can read documentation on how to download the encryption, signing and certificate bundle? (or does it have to be embedded in the content)? FYI @benjamin-mortimer - happy to chat IRL

File: source/create-a-network-connection.html.md.erb [hunk 2]
```diff
@@ -14,21 +14,19 @@ The DCS and your service use mTLS to prove to each other that they possess the p
 Before setting up mTLS, you’ll need:
 
 * to [generate a private key and request a GDS-signed certificate](/generate-keys-and-request-certificates) - it can take up to 2 working days for the DCS to issue a certificate
-* the DCS Certificate Authority (CA) bundle - [request it from the DCS pilot helpdesk](/support) if you have not received it already
+* the [DCS Certificate Authority (CA) certificate bundle]
```
Comment (PippaClarkGDS) [inline source/create-a-network-connection.html.md.erb:17]:
```suggestion
* to download the [DCS Certificate Authority (CA) certificate bundle]
```

File: source/access-the-data-in-a-DCS-response.html.md.erb [hunk 3]
```diff
@@ -29,8 +29,8 @@ You’ll need to use a library to work with the JSON Web Signature (JWS) and JWE
 
 You’ll need:
 
-* [your encryption key](/generate-keys-and-request-certificates/#generate-a-private-rsa-key)
-* [the DCS public signing certificate](/generate-keys-and-request-certificates/#create-a-certificate-signing-request-csr)
+* [your encryption private key](/generate-keys-and-request-certificates/#generate-a-private-rsa-key)
+* the [DCS public signing certificate]
```
Comment (PippaClarkGDS) [inline source/access-the-data-in-a-DCS-response.html.md.erb:33]:
```suggestion
* to download the [DCS public signing certificate]
```

File: source/integrating-with-the-dcs.html.md.erb (hunk 1 above)
Comment (PippaClarkGDS) [reply source/integrating-with-the-dcs.html.md.erb:33]: we would also usually make the link active ... eg if the user had to download the cert, we would include the 'Download' instruction in the hyperlink so it's clear what the link does. Is it possible to do this? It's for accessibility reasons so users using screen readers don't just hear 'DCS public signing certificate' when hearing the links read out to them @benjamin-mortimer

**Comment by bjgill** [comment]
@PippaClarkGDS - thanks for that. I've made all the links active or removed them.

And I've moved the addition to the numbered list somewhere that seems less disruptive.

File: source/access-the-data-in-a-DCS-response.html.md.erb [hunk 4]
```diff
@@ -86,7 +86,7 @@ To validate the inner signature, you must:
 
 1. Use your JWS library.
 1. Pass your JWS library the decrypted string you generated in the previous step.
-1. Pass your JWS library the DCS signing certificate.
+1. Pass your JWS library the DCS public signing certificate.
```
Comment (PippaClarkGDS) [inline source/access-the-data-in-a-DCS-response.html.md.erb:89]:
```suggestion
1. [Pass your JWS library the DCS public signing certificate][DCS public signing certificate].
```

File: source/access-the-data-in-a-DCS-response.html.md.erb [hunk 5]
```diff
@@ -52,7 +52,7 @@ To validate the outer signature, you must:
 
 1. Use your JWS library’s signature validation method.
 1. Pass your JWS library the DCS response body string.
-1. Pass your JWS library the DCS signing certificate.
+1. Pass your JWS library the DCS public signing certificate.
```
Comment (PippaClarkGDS) [inline source/access-the-data-in-a-DCS-response.html.md.erb:55]:
```suggestion
1. [Pass your JWS library the DCS public signing certificate][DCS public signing certificate].
```

**Pull request merged**